### PR TITLE
"Modify the files containing AGPL 3.0 license comments, changing the …

### PR DESCRIPTION
**Pull request by SunHappyboy**
…comments to reflect the MPL v2 license instead."

*A good pull request has the following information:*
- A reference to a related issue in your repository.
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes. ( optional )

### CODE REVIEW

**Review by nolankramer — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Add 'repo' label to any root file changes
 repo:
-- '*'
-
+- owner:SunHappyboy
```
Comment (ZigRazor) [inline .github/labeler.yml:3]: Please remove this autoreference from this file

Comment (ZigRazor) [reply .github/labeler.yml:3]: Please correct it

**Review by ZigRazor — CHANGES_REQUESTED** [review]

**Comment by SunHappyboy** [comment]
I ras revert

File: .github/labeler.yml (hunk 1 above)
Comment (SunHappyboy) [reply .github/labeler.yml:3]: I has remove it ;

**Review by ZigRazor — APPROVED** [review]

**Pull request merged**